portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -12,21 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Date;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private List<String> placesLived = new ArrayList<>();
+  
+  @Override
+  public void init() {
+    placesLived.add("Canmore, Alberta");
```

Comment:
Can you do this during the creation of the list, and make the list a final const?

ex. = new ArrayList<String>(Arrays.asList(...));

Reply:
Done